docs/operator-manual/upgrading/2.6-2.7.md

```diff
@@ -81,3 +81,10 @@ Read the full [documentation](../deep_links.md) to see all possible combinations
 Argo CD now supports the `helm.sh/resource-policy` annotation to control the deletion of resources. The behavior is the same as the behavior of
 `argocd.argoproj.io/sync-options: Delete=false` annotation: if the annotation is present and set to `keep`, the resource will not be deleted
 when the application is deleted.
+
+## Check your Kustomize patches for `--redis` changes
+
+Starting in Argo CD 2.7, the Redis server name is no longer passed via the `--redis` flag. If your environment uses 
```

Comment:
I think this needs to be better phrased as the flag is still there:
https://github.com/argoproj/argo-cd/blob/master/util/cache/cache.go#L89